src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksWebServiceCallsServiceImpl.java

```diff
@@ -114,8 +115,17 @@ private URI buildPaymentWorksApprovedNewVendorRequestsURI() {
     
     @Override
     public PaymentWorksVendor obtainPmwNewVendorRequestDetailForPmwIdentifier(String pmwNewVendorRequestId, PaymentWorksNewVendorRequestsBatchReportData reportData) {
-        PaymentWorksNewVendorRequestDetailDTO pmwDetailForSpecificVendorDTO = retrieveAllPaymentWorksDetailsForRequestedVendor(buildPaymentWorksPendingNewVendorRequestDetailURI(pmwNewVendorRequestId));
-        PaymentWorksVendor pmwDetailForSpecificVendor = getPaymentWorksDtoToPaymentWorksVendorConversionService().createPaymentWorksVendorFromPaymentWorksNewVendorRequestDetailDTO(pmwDetailForSpecificVendorDTO, reportData);
+        PaymentWorksNewVendorRequestDetailDTO pmwDetailForSpecificVendorDTO = null;
+        PaymentWorksVendor pmwDetailForSpecificVendor = null;
+        try {
+            pmwDetailForSpecificVendorDTO = retrieveAllPaymentWorksDetailsForRequestedVendor(buildPaymentWorksPendingNewVendorRequestDetailURI(pmwNewVendorRequestId));
+        } catch (ResponseProcessingException rpe) {
```

Comment:
Is the exception type being caught too specific? Perhaps it should catch a broader exception type instead (like RuntimeException or Exception).

Reply:
I was using the example exception in the Jira, I can broaden it.